packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -2063,7 +1983,10 @@ function commitRootImpl(root, renderPriorityLevel) {
     rootDoesHavePassiveEffects = false;
     rootWithPendingPassiveEffects = root;
     pendingPassiveEffectsLanes = lanes;
-    pendingPassiveEffectsRenderPriority = renderPriorityLevel;
+    pendingPassiveEffectsRenderPriority =
+      renderPriorityLevel === NoLanePriority
+        ? DefaultLanePriority
+        : renderPriorityLevel;
```

Comment:
I switched this over to use the updateLanePriority. Plan is to remove this in a follow up.